qt/DetailsDialog.cc

```diff
@@ -632,6 +663,25 @@ void DetailsDialog::refreshUI()
 
     ui_.uploadedValueLabel->setText(string);
 
+    //  myUploadedThisSessionLabel
+    if (torrents.empty())
+    {
+        string = none;
+    }
+    else
+    {
+        auto uploaded_this_session = uint64_t{};
+
+        for (Torrent const* const t : torrents)
+        {
+            uploaded_this_session += t->uploadedThisSession();
+        }
+
+        string = tr("%1").arg(Formatter::storage_to_string(uploaded_this_session));
```

Comment:
`tr("%1")` seems unnecessary, are translators expected to ever modify this string?